index.html

```diff
@@ -180,14 +180,15 @@
         </li>
         <li>When the `type` attribute of the [link] element of a <a>preload
         link</a> that is already [in a Document] but was previously not
-        obtained due to the `type` attribute specifying an unsupported
+        obtained due to the `type` attribute not specifying a
+        [parsable MIME type] or specifying an [unsupported MIME type] for the
         <a>request destination</a> is set, removed, or changed.
         </li>
         <li>When the `media` attribute of the [link] element of a <a>preload
-        link</a> that is already [in a Document] contains a [valid media query
-        list] that [matches the environment] of the user. Otherwise, if the
-        media query is invalid, or [evaluates to false][] [[!mediaqueries-4]],
-        the user agent SHOULD NOT obtain the resource.
+        link</a> that is already [in a Document] but was not previousely
+        obtained due the `media` attribute's value being not a
+        [valid media query list] or one that [evaluates] to false is set,
+        removed, or changed.
```

Comment:
So, I looked at that and couldn't find "matches the user's environment" in https://drafts.csswg.org/mediaqueries/ (so didn't really know if it means that the MQ list evaluates to true).

Did MQs change and that definition needs to be updated? Or is it defined elsewhere?